src/themes/default.tsx

```diff
@@ -0,0 +1,4 @@
+import { css } from 'styled-components'
+export const defaultTheme = css`
+            --color-primary: #5cb85c;
```

Comment:
Since you are already defining variables here, and you are already using mixin above `const containerPadding = '15px';` - dont you also want to create a theme that will include spacings, colors, radii etc, and a set of mixins (that will potentially reduce bundle size, like tailwind)?